lib/connection.js

```diff
@@ -717,6 +717,13 @@ class Connection extends EventEmitter {
   _remoteCallbackWrapper(messageId, error, ...result) {
     this._callback(messageId, error, result);
   }
+
+  _log(message, ...args) {
+    this.emit(message, ...args);
+    if (this.server) {
+      this.server.emit(message, ...args, this);
```

Comment:
I think connection argument should be second and not the last, as this will ease using a general listener that'll get a connection as a second parameter and not extract it from the array of all args.